Debug App/Sources/Network/Networking.swift

```diff
@@ -100,27 +100,27 @@ class Networking {
             request.httpBody = body
             
             let bodyJson = try? JSONSerialization.jsonObject(with: body, options: .allowFragments)
-            msg += "Body:\n\(bodyJson ?? [:])\n"
+//            msg += "Body:\n\(bodyJson ?? [:])\n"
         }
 
-        print(msg)
+        Primer.shared.logger?.debug(message: msg)
         msg = ""
         
         URLSession.shared.dataTask(with: request, completionHandler: { (data, response, err) in
             DispatchQueue.main.async {
-                msg += "RESPONSE\n"
-                msg += "URL: \(request.url?.absoluteString ?? "Invalid")\n"
+//                msg += "RESPONSE\n"
+//                msg += "URL: \(request.url?.absoluteString ?? "Invalid")\n"
```

Comment:
These items which are commented out: leftover from development, or do we want to remove them entirely>

Reply:
These are to be removed 👍 Generally, I've tried to avoid multiline logs as they don't present well in the console + we should try to keep logs discrete so they are more easily filterable in the future